README.md

```diff
@@ -1,7 +1,7 @@
 # ecAudit
 
-[![tests](https://github.com/Ericsson/ecaudit/actions/workflows/test.yml/badge.svg?branch=release/c3.0)](https://github.com/Ericsson/ecaudit/actions/workflows/test.yml?query=branch%3Arelease/c3.0)
-[![coverage](https://codecov.io/gh/ericsson/ecaudit/branch/release/c3.0/graph/badge.svg)](https://codecov.io/gh/ericsson/ecaudit/branch/release/c3.0)
+[![tests](https://github.com/Ericsson/ecaudit/actions/workflows/test.yml/badge.svg?branch=release/c3.11)](https://github.com/Ericsson/ecaudit/actions/workflows/test.yml?query=branch%3Arelease/c3.11)
```

Comment:
This is not correct right? You want to merge this PR into 3.0 branch but you change these to 3.11?

Reply:
I try to keep these files the same on all branches so before all were linking to 3.0 but since 3.11 is the default branch and the oldest we do active development on, 3.0 is maintenance only, it seamed better to link to 3.11. But we could change and link to the selected branch instead. I don't know what would be least confusing.

Reply:
After double checking I realize that I'm wrong and you are right, my mistake. I will update revert that change.

Reply:
Now it should be correct, thanks for pointing out my error.